app/src/main/java/org/mozilla/fenix/home/HomeFragment.kt

```diff
@@ -198,13 +202,13 @@ class HomeFragment : Fragment() {
         inflater: LayoutInflater,
         container: ViewGroup?,
         savedInstanceState: Bundle?
-    ): View? {
-        val view = inflater.inflate(R.layout.fragment_home, container, false)
+    ): View {
+        _binding = FragmentHomeBinding.inflate(inflater)
```

Comment:
`container` might be needed here.

Reply:
Thanks! Was going with the idea that if this layout is not added to the parent there's no need for the container but it's probably best to pass this to ensure the same as before layoutParams are generated.